cmake/modules/packages/FindPoppler.cmake

```diff
@@ -71,82 +71,82 @@ set(Poppler_Qt4_header "poppler-qt4.h")
 set(Poppler_Glib_header "poppler.h")
 
 foreach(_comp IN LISTS Poppler_FIND_COMPONENTS)
-  set(POPPLER_${_comp}_FOUND FALSE)
+  set(Poppler_${_comp}_FOUND FALSE)
 endforeach()
 
 foreach(_comp IN LISTS Poppler_known_components)
   list(FIND Poppler_FIND_COMPONENTS "${_comp}" _nextcomp)
   if(_nextcomp GREATER -1)
-    find_path(POPPLER_${_comp}_INCLUDE_DIR
+    find_path(Poppler_${_comp}_INCLUDE_DIR
               NAMES ${Poppler_${_comp}_header}
               PATH_SUFFIXES poppler
               HINTS ${PC_Poppler_${_comp}_INCLUDE_DIRS})
-    find_library(POPPLER_${_comp}_LIBRARY
+    find_library(Poppler_${_comp}_LIBRARY
             NAMES ${Poppler_${_comp}_lib}
             HINTS ${PC_Poppler_${_comp}_LIBRARY_DIRS})
```

Comment:
This doesn't have to be handled in this PR:
AFAICS, `PC_Poppler_${_comp}_INCLUDE_DIRS}` and `${PC_Poppler_${_comp}_LIBRARY_DIRS}` won't be initialized in this file. I assume that the component pc files were meant to be loaded in another loop with, the `PC_Poppler_${_comp}` prefix.

Reply:
maybe... I didn't write this.  For GDAL purposes, we don't use the components functionality when looking for Poppler